.gitignore

```diff
@@ -1,4 +1,6 @@
-Resources/doc/_build
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
-vendor
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore